_posts/2024-02-06_openttd-truetype-fonts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TrueType fonts for OpenTTD
+author: Zephyris
+---
+
+The pixel-art OpenTTD fonts are iconic, but were designed for 640x480 CRT monitors rather than today's 4k monsters.
+The pixel-perfect rendering and 2x or 4x upscaling of the old bitmap/sprite-based font gives jagged and hard-to-read text, especially as they were designed to be horizontally smoothed by CRT scanlines.
+How do you update a 30-year old font to make it more suitable for today?
```

Comment:
This could be a good first sentence for the whole story, I think. Although maybe 30 years isn't old for a font. 😛